src/sidebar/templates/sidebar-content.html

```diff
@@ -36,7 +36,7 @@
 <sidebar-content-error
   class="sidebar-content-error"
   logged-out-error-message="'This group is not available.'"
-  logged-in-error-message="'You either do not have permission to view this group, the group does not exist, or the group is not visible at this URL.'"
+  logged-in-error-message="'This group is not available.'"
```

Comment:
I discussed this with @lyzadanger earlier today. She said just making them the same message would be fine in her opinion.